20-kustomize/02-fullstack/pingintelligence/env_vars.pingintelligence

```diff
@@ -0,0 +1 @@
+PING_IDENTITY_ACCEPT_EULA=YES
```

Comment:
Do we need this and the configMapGenerator? Since this var is already in the standalone configMap I don't think that it's necessary.